scraper.rb

```diff
@@ -2,6 +2,27 @@
 require 'mechanize'
 
 # Scraping from Masterview 2.0
+case ENV['MORPH_PERIOD']
```

Comment:
This seems incredibly complex. I think it would be a good challenge for you to refactor and simplify it @LoveMyData